### PR TITLE
Fixed navbar and set up firebase emulator.

### PR DESCRIPTION
**Pull request by phonemhkyaw**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -46,30 +49,26 @@ const App = (props) => {
...
-          <Route path="blogs" element={<Blogs />} />
-          <Route path="contact" element={<Contact />} />
-          <Route path="*" element={<NoPage />} />
-        </Route>
-      </Routes>
-</BrowserRouter>
- ignore this just reference
-*/
\ No newline at end of file
+//  <BrowserRouter>
```
Comment (kkaungyyang) [inline src/App.js:65]: Do you need to include these comments?

Comment (phonemhkyaw) [reply src/App.js:65]: i will remove it in the next PR

File: src/components/home.jsx [hunk 2]
```diff
@@ -28,8 +32,8 @@ const Home = () => {
         e.preventDefault();
         const querySnapshot = await getDocs(collection(db, "messages"));
         querySnapshot.forEach((doc) => {
-            console.log(doc.data().message);
-            return (doc.data().message);
+            console.log(doc.data().message, Number(doc.data().number));
```
Comment (kkaungyyang) [inline src/components/home.jsx:35]: console logs should be removed if you don't need them

File: src/components/navbar.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+        </Button>
+        <Dropdown
+          menu={{
+            items,
+          }}
+          placement="bottomRight"
+        >
+          <Button>bottomRight</Button>
+        </Dropdown>
+        {/* <Dropdown
```
Comment (kkaungyyang) [inline src/components/navbar.js:104]: remove unneeded comment

**Review by kkaungyyang — APPROVED** [review]
LGTM! please address my ask, approved right after

File: src/components/home.jsx (hunk 2 above)
Comment (phonemhkyaw) [reply src/components/home.jsx:35]: ok. noted

File: src/components/navbar.js (hunk 3 above)
Comment (phonemhkyaw) [reply src/components/navbar.js:104]: ok, next pr!

**Pull request merged**